config/samples/hybris_v1alpha1_hybrisapp.yaml

```diff
@@ -4,7 +4,13 @@ metadata:
   name: hybrisapp-sample
 spec:
   baseImageName: hybris-base
-  baseImageTag: v0.1
+  baseImageTag: v0.2
   sourceRepoRef: master
-  sourceRepoURL: https://github.com/redhat-sap/sap-commerce-app-example.git
+  sourceRepoURL: https://github.com/faizalak/sap-commerce-app-example.git
```

Comment:
I don't think personal information should be used in the examples.

Reply:
I will update this